res/xml/preferences.xml

```diff
@@ -164,6 +164,10 @@
 				android:defaultValue="false"
 				android:title="@string/settings_auth_full_uri"
 				android:summary="@string/settings_auth_full_uri_info"/>
+		<CheckBoxPreference
+			android:key="publish_enable"

```

Comment:
maybe this should be done on a per-account basis instead of a global setting?  @saghul what do you think?